net.sf.eclipsecs.checkstyle/pom.xml

```diff
@@ -9,8 +9,42 @@
     <artifactId>net.sf.eclipsecs.checkstyle</artifactId>
     <packaging>eclipse-plugin</packaging>
     <name>Checkstyle Core Library Plugin</name>
+    
+    <properties>
+        <checkstyle.version>8.33</checkstyle.version>
+    </properties>
     <build>
         <plugins>
+            <plugin>
+                <groupId>com.googlecode.maven-download-plugin</groupId>
+                <artifactId>download-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>download-checkstyle-jar</id>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                        <configuration>
+                            <skip>true</skip><!-- jar has not the same content as the one present. maybe it is built manually!? -->
```

Comment:
Can you only take care of the zip file in this PR? I don't think downloading the jar dependency using download-maven-plugin will ever work.

Reply:
Done.